.github/workflows/nix.yml

```diff
@@ -0,0 +1,24 @@
+name: Ensure nix-dev environment still builds correctly
```

Comment:
maybe shorten the name so that it doesn't overflow in the interface?

Reply:
Sure, got a suggestion I can click "add" on here in review comment? I'm currently away from pc (on smartphone device you see).

Reply:
```suggestion
name: Verify nix environment
```